pom.xml

```diff
@@ -745,7 +745,7 @@
         <org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.version>2.4.21</org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.version>
         <org.wso2.carbon.logging.service.version>4.10.6</org.wso2.carbon.logging.service.version>
         <org.wso2.carbon.event.publisher.version>5.2.15</org.wso2.carbon.event.publisher.version>
-        <identity.branding.preference.management.version>1.0.4</identity.branding.preference.management.version>
+        <identity.branding.preference.management.version>1.0.6-SNAPSHOT</identity.branding.preference.management.version>
```

Comment:
update before merging the PR

Reply:
Updated with https://github.com/wso2/identity-api-server/pull/495/commits/f9589093cec6472b487f42653671b90b1c7c8819